mythril/src/vm.rs

```diff
@@ -466,13 +528,13 @@ impl VirtualMachine {
         vcpu: &crate::vcpu::VCpu,
         responses: &mut ResponseEventArray,
     ) -> Result<()> {
-        let dev = self
-            .config
-            .virtual_devices()
-            .find_device(ident)
-            .ok_or_else(|| {
-                Error::MissingDevice("Unable to dispatch event".into())
-            })?;
+        let dev = match self.virtual_device_map.find_device(ident) {
+            Some(dev) => dev,
+            None => {
+                // warn!("Unable to dispatch event");
```

Comment:
Could this be changed to a TODO? The warn could also be useful

Reply:
Made this back into an error for non-portio devices